overlord/snapstate/storehelpers.go

```diff
@@ -239,29 +237,35 @@ func installInfo(ctx context.Context, st *state.State, name string, revOpts *Rev
 		InstanceName: name,
 	}
 
-	// TODO: support ignore-validation
-	enforcedSets, err := EnforcedValidationSets(st)
-	if err != nil {
-		return store.SnapActionResult{}, err
+	if flags.IgnoreValidation {
```

Comment:
Should there be an `else` case setting the flags to `SnapActionEnforceValidation`, like there is in the update path? AFAICT, not setting the flag isn't the same as setting it to Enforce since the corresponding JSON field is a *bool with omitempty. Not sure if that matters in this case, just thought I'd ask

Reply:
the default is not to ignore so either way is fine